src/backend/reactions/reaction.cpp

```diff
@@ -54,10 +57,12 @@ namespace big
 	}
 
 
-	void reaction::process(player_ptr player)
+	void reaction::process(player_ptr player, bool is_crash_exclude_TSE)
```

Comment:
Every call to `reaction::process` you're hard coding the value `true/false`.

Perhaps this value should be passed in the constructor of the reaction?
https://github.com/YimMenu/YimMenu/blob/da028f03b3120a50de5e93bf1d4bc9a07fe9d3f8/src/core/settings.hpp#L176-L209
https://github.com/YimMenu/YimMenu/blob/da028f03b3120a50de5e93bf1d4bc9a07fe9d3f8/src/backend/reactions/reaction.hpp#L25

To keep it readable you could either choose a logical default in the constructor.

```cpp
reaction(const char* event_name, const char* notify_message, const char* announce_message, bool is_crash_exclude_tse = false /* true */ );
```

Reply:
It might just be me but the `is_crash_exclude_TSE` variable is quite confusing on what it does, could you clarify on what it means exactly? That way we could search for a different name.

Reply:
false: script event crash and all other infraction
true: exclude other crashes except for script event crash